plugins/Diagnostics/Controller.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics;
+
+use Piwik\Config;
+use Piwik\Piwik;
+use Piwik\View;
+use Piwik\Settings;
+
+/**
+ * A controller let's you for example create a page that can be added to a menu. For more information read our guide
+ * http://developer.piwik.org/guides/mvc-in-piwik or have a look at the our API references for controller and view:
+ * http://developer.piwik.org/api-reference/Piwik/Plugin/Controller and
+ * http://developer.piwik.org/api-reference/Piwik/View
+ */
```

Comment:
I'm not sure about that. It might make it easier for when any developers have a look at it and try to understand it or when they want to know more about it. I'm always not sure myself what to do. Sometimes I remove them, sometimes I don't :) Currently I tend to rather have them as it is kinda nice to have such documentation everywhere. Problem is when the links are outdated etc.